index.js

```diff
@@ -1,148 +1,211 @@
-var marked = require('marked');
+const marked = require('marked');
 marked.setOptions({
-	breaks: true,
-	smartyPants: true
+  breaks: true,
+  smartyPants: true
 });
 
-var J2M = function() {};
+const J2M = function () { };

```

Comment:
not a big thing, can we put fns: `md_to_html ` & `jira_to_html ` & `to_jira` & `to_markdown`  into the `{ }` on line 7, or even better if possible then make `J2M` to be a `class`, this could be more readable.

Reply:
Good call. Addressed in https://github.com/karimatthews/J2M/pull/1/commits/ba965283e2bccdf6c5d1f347fd7ca6defc0851cb

Reply:
nice one, thanks @karimatthews